library/options.json

```diff
@@ -28,6 +28,9 @@
     }
   },
   "render": {
+    "reader": {
+      "type": "string"
+    },
```

Comment:
I dont think you need an option

Reply:
maybe you do, let me think

Reply:
indeed, you dont need an option for this, 

`factory::getReader(filename, forceReader)` may be a better way to do this